docs/streams.html

```diff
@@ -790,13 +790,15 @@
         </p>
         <br>
 
-        <h2><a id="streams_upgrade" href="#streams_upgrade">Upgrade Guide and API Changes</a></h2>
+        <h2><a id="streams_upgrade_and_api" href="#streams_upgrade_and_api">Upgrade Guide and API Changes</a></h2>
 
         <p>
         See the <a href="/documentation/#upgrade_1020_streams">Upgrade Section</a> for upgrading a Kafka Streams Application from 0.10.1.x to 0.10.2.0.
+        It highlights incompatible changes you need to consider to upgrade your code and application.
```

Comment:
In line 796 above, I think `"/documentation/#upgrade_1020_streams"` would redirect correctly. cc @derrickdoo .